lib/mimic/cover.ex

```diff
@@ -11,12 +11,21 @@ defmodule Mimic.Cover do
     :cover.is_compiled(module) != false
   end
 
+  @doc false
+  def private_functions_exported? do
+    function_exported?(:cover, :get_term, 1)
+  end
+
   @doc false
   def export_private_functions do
-    {_, binary, _} = :code.get_object_code(:cover)
-    {:ok, {_, [{_, {_, abstract_code}}]}} = :beam_lib.chunks(binary, [:abstract_code])
-    {:ok, module, binary} = :compile.forms(abstract_code, [:export_all])
-    :code.load_binary(module, '', binary)
+    if not private_functions_exported?() do
+      {_, binary, _} = :code.get_object_code(:cover)
+      {:ok, {_, [{_, {_, abstract_code}}]}} = :beam_lib.chunks(binary, [:abstract_code])
+      {:ok, module, binary} = :compile.forms(abstract_code, [:export_all])
+      {:module, :cover} = :code.load_binary(module, '', binary)
```

Comment:
this could potentially return an error tuple, which would have been ignored. do we want to keep this change?

Reply:
We want this to fail if it happens so I think it's fine to expect this as the feature won't work without these calls.